blog/templates/blog/post_list.html

```diff
@@ -0,0 +1,24 @@
+{% load i18n %}
+
+<html>
+<head>
+    <title>{% trans "Django Girls blog" %}</title>
+</head>
+<body>
+<div>
+    <h1><a href="/">{% trans "Django Girls Blog" %}</a></h1>
+</div>
+
+<div>
+    <p>{% trans "published: 14.06.2014, 12:14" %}</p>
+    <h2><a href="">{% trans "My first post" %}</a></h2>
+    <p>{% trans "Aenean eu leo quam. Pellentesque ornare sem lacinia quam venenatis vestibulum. Donec id elit non mi porta gravida at eget metus. Fusce dapibus, tellus ac cursus commodo, tortor mauris condimentum nibh, ut fermentum massa justo sit amet risus." %}</p>
```

Comment:
Fun fact: there's a special tag for generating fish-text in template engines :)

Reply:
https://docs.djangoproject.com/en/2.0/ref/templates/builtins/#lorem